taichi/runtime/llvm/node_dense.h

```diff
@@ -2,41 +2,21 @@
 
 // Specialized Attributes and functions
 struct DenseMeta : public StructMeta {
-  bool bitmasked;
   int morton_dim;
 };
 
-STRUCT_FIELD(DenseMeta, bitmasked)
 STRUCT_FIELD(DenseMeta, morton_dim)
 
 i32 Dense_get_num_elements(Ptr meta, Ptr node) {
   return ((StructMeta *)meta)->max_num_elements;
 }
 
 void Dense_activate(Ptr meta, Ptr node, int i) {
```

Comment:
Where this func is called? Can we remove them completely?

Reply:
No, see `SNodeLookupStmt`

Reply:
Around here

https://github.com/taichi-dev/taichi/blob/1371a07313d6258f9de12c67a4bc013dfef81476/taichi/codegen/codegen_llvm.h#L1145-L1151